services/logs/logs.go

```diff
@@ -43,27 +46,39 @@ type File struct {
 type Log struct {
 	FilePath string
 	UnitName string
+	Command  string
 }
 
 var DefaultLogs = []Log{
-	{"/var/log/consul.log", "consul"},
-	{"/var/log/createdb.log", ""},
-	{"/var/log/cron.log", "crond"},
-	{"/var/log/dashboard-upgrade.log", ""},
-	{"/var/log/grafana/grafana.log", ""},
-	{"/var/log/mysql.log", ""},
-	{"/var/log/mysqld.log", "mysqld"},
-	{"/var/log/nginx.log", "nginx"},
-	{"/var/log/nginx/access.log", ""},
-	{"/var/log/nginx/error.log", ""},
-	{"/var/log/node_exporter.log", "node_exporter"},
-	{"/var/log/orchestrator.log", "orchestrator"},
-	{"/var/log/pmm-manage.log", "pmm-manage"},
-	{"/var/log/pmm-managed.log", "pmm-managed"},
-	{"/var/log/prometheus1.log", "prometheus1"},
-	{"/var/log/prometheus.log", "prometheus"},
-	{"/var/log/qan-api.log", "percona-qan-api"},
-	{"/var/log/supervisor/supervisord.log", ""},
+	{"/var/log/consul.log", "consul", ""},
+	{"/var/log/createdb.log", "", ""},
+	{"/var/log/cron.log", "crond", ""},
+	{"/var/log/dashboard-upgrade.log", "", ""},
+	{"/var/log/grafana/grafana.log", "", ""},
+	{"/var/log/mysql.log", "", ""},
+	{"/var/log/mysqld.log", "mysqld", ""},
+	{"/var/log/nginx.log", "nginx", ""},
+	{"/var/log/nginx/access.log", "", ""},
+	{"/var/log/nginx/error.log", "", ""},
+	{"/var/log/node_exporter.log", "node_exporter", ""},
+	{"/var/log/orchestrator.log", "orchestrator", ""},
+	{"/var/log/pmm-manage.log", "pmm-manage", ""},
+	{"/var/log/pmm-managed.log", "pmm-managed", ""},
+	{"/var/log/prometheus1.log", "prometheus1", ""},
+	{"/var/log/prometheus.log", "prometheus", ""},
+	{"/var/log/qan-api.log", "percona-qan-api", ""},
+	{"/var/log/supervisor/supervisord.log", "", ""},
+	{"/etc/prometheus.yml", "expand", "/usr/bin/cat"},
+	{"/etc/supervisord.d/pmm.ini", "expand", "/usr/bin/cat"},
+	{"/etc/nginx/conf.d/pmm.conf", "expand", "/usr/bin/cat"},
+	{"prometheus_targets.txt", "expand", "/usr/bin/curl -s http://localhost/prometheus/targets"},
+	{"consul_nodes.txt", "expand", "/usr/bin/curl -s http://localhost/v1/internal/ui/nodes?dc=dc1"},
+	{"qan-api_instances.txt", "expand", "/usr/bin/curl -s http://localhost/qan-api/instances"},
+	{"managed_RDS-Aurora.txt", "expand", "/usr/bin/curl -s http://localhost/managed/RDS"},
+	{"pmm-version.txt", "expand", "head -1 /srv/update/main.yml"},
```

Comment:
PMM version is defined in `main.go` and can be passed as an argument to service.

Reply:
for now this is several places with version  and it's better to keep this method in place too